src/main/java/net/sigmalab/maven/plugin/jira/AbstractJiraMojo.java

```diff
@@ -98,7 +100,7 @@ void loadUserInfoFromSettings() {
          * for the project.
          */
         if ( getJiraProjectKey() == null ) {
```

Comment:
**Assign this magic number 8 to a well-named constant, and use the constant instead.**

A magic number is a number that comes out of nowhere, and is directly used in a statement. Magic numbers are often used, for instance to limit the number of iterations of a loops, to test the value of a property, etc.

Using magic numbers may seem obvious and straightforward when you're writing a piece of code, but they are much less obvious and straightforward at debugging time.

That is why magic numbers must be demystified by first being assigned to clearly named variables before being used.